docs/section16/Rule 16-5.md

```diff
@@ -22,10 +22,9 @@ N/A
 - For each elevator in the baseline RMD: `for elevator in B_RMD...elevators`
   - Get the modeled schedule for the baseline elevator cab ventilation fan: `cab_ventilation_schedule_b = elevator.cab_ventilation_fan_multiplier_schedule`
   - Get the modeled schedule for the baseline elevator cab lighting: `cab_lighting_schedule_b = elevator.cab_lighting_multiplier_schedule`
-  - Compare the schedules: `comparison_data = compare_schedules(cab_lighting_schedule_b, cab_ventilation_schedule_b):`  
 
   **Rule Assertion:**  
-    - Case 1: If the schedules for lighting and ventilation are identical and continuous for all hours: `if comparison_data['SCHEDULE_1_EFLH'] == len(cab_lighting_schedule_b) == len(cab_ventilation_schedule_b): PASS`
+    - Case 1: If the schedules for lighting and ventilation are identical and continuous for all hours: `if sum(cab_ventilation_schedule_b) == sum(cab_lighting_schedule_b) == len(cab_lighting_schedule_b) == len(cab_ventilation_schedule_b): PASS`
```

Comment:
I may be unclear on the definition of “operate continuously”, but it seems that the elevator cab lights & ventilation should operate at 100% during all building operation hours, so would be something like 24hr / 7 for hotel. I think there’s a building operation schedule that these schedules can be checked against

Reply:
Operate continuously is assumed to mean 8760/8784 EFLH here. I don't think there is a definition that we can point to that includes "during operating hours" or anything to that effect but let me know if I am wrong.

Reply:
Is the length of the schedule always 8760 or 8784?

Reply:
Any time the compare_schedules function is used, it includes an assertion that the schedules are the same length as the total year hours. (That function is not used here, but I am using that as an example where that assumption is worked into the code) I don't believe there is a validation check for this, so the RPD file would still be valid if schedules are not 8760/8784, but they would fail rule evaluations where they are expected to be 8760/8784

Reply:
Ok. I just want to make sure that if we’re expecting it to be 8760/8784, that it actually is. If you’re confident that it is based on the current logic, then I’m ok with that.